lib/streamlit/elements/widgets/button.py

```diff
@@ -337,6 +338,75 @@ def download_button(
             ctx=ctx,
         )
 
+    @gather_metrics("link_button")
+    def link_button(
+        self,
+        label: str,
+        url: str,
+        *,
+        help: Optional[str] = None,
+        type: Literal["primary", "secondary"] = "secondary",
+        disabled: bool = False,
+        use_container_width: bool = False,
+    ) -> "DeltaGenerator":
+        r"""Display a link button element.
+
+        Parameters
+        ----------
+        label : str
+            A short label explaining to the user what this button is for.
+            The label can optionally contain Markdown and supports the following
+            elements: Bold, Italics, Strikethroughs, Inline Code, and Emojis.
+
+            This also supports:
+
+            * Emoji shortcodes, such as ``:+1:``  and ``:sunglasses:``.
+              For a list of all supported codes,
+              see https://share.streamlit.io/streamlit/emoji-shortcodes.
+
+            * LaTeX expressions, by wrapping them in "$" or "$$" (the "$$"
+              must be on their own lines). Supported LaTeX functions are listed
+              at https://katex.org/docs/supported.html.
+
+            * Colored text, using the syntax ``:color[text to be colored]``,
+              where ``color`` needs to be replaced with any of the following
+              supported colors: blue, green, orange, red, violet, gray/grey, rainbow.
+
+            Unsupported elements are unwrapped so only their children (text contents) render.
+            Display unsupported elements as literal characters by
+            backslash-escaping them. E.g. ``1\. Not an ordered list``.
+        url : str
+            The url to be opened on user click
+        help : str
+            An optional tooltip that gets displayed when the button is
+            hovered over.
+        type : "secondary" or "primary"
+            An optional string that specifies the button type. Can be "primary" for a
+            button with additional emphasis or "secondary" for a normal button. This
+            argument can only be supplied by keyword. Defaults to "secondary".
+        disabled : bool
+            An optional boolean, which disables the download button if set to
+            True. The default is False. This argument can only be supplied by
```

Comment:
nit: I think you can remove the `This argument can only be supplied by keyword.`. We still have that scattered around in some old docstrings, but we left out this information for the more recent docstrings. I think the docs team is planning to unify/clean that up a soon across our codebase.

Reply:
Done